test/system/tests/quote.sh

```diff
@@ -51,14 +51,7 @@ Handle_ak_quote=0x81010016
 Handle_ek_quote=0x81010017
```

Comment:
Can you trim the quoted line in the commit message, we get the idea. No need to have such a long hex number :)